subprojects/launcher/src/integTest/groovy/org/gradle/launcher/continuous/DeploymentContinuousBuildIntegrationTest.groovy

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright 2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.launcher.continuous
+
+class DeploymentContinuousBuildIntegrationTest extends Java7RequiringContinuousIntegrationTest {
+    File triggerFile = file('triggerFile')
+    File keyFile = file('keyFile')
+
+    def setup() {
+        buildFile << """
```

Comment:
Can you pull this into a test fixture to share this with the other test? This looks a lot like DeploymentHandleContinuousBuildCrossVersionSpec's code?

The trigger/key files above would be part of the fixture too and the assert method below.

Reply:
Done.